.gitignore

```diff
@@ -19,3 +20,5 @@
 
 # Go workspace file
 go.work
+
+.idea/
```

Comment:
nit: since the project doesnt introduce this dir it should be configured in the  local env .gitignore.

Reply:
what is local env .gitignore?

Reply:
Its similar to the project gitignore but affects whole local environment, here is a nice gist that explain what is it and  how to work with it https://gist.github.com/subfuzion/db7f57fff2fb6998a16c